x-pack/solutions/security/plugins/security_solution/public/explore/network/components/embeddables/embedded_map.tsx

```diff
@@ -37,36 +37,30 @@ interface EmbeddableMapProps {
   maintainRatio?: boolean;
 }
 
-const EmbeddableMapRatioHolder = styled.div.attrs(() => ({
-  className: 'siemEmbeddable__map',
-}))<EmbeddableMapProps>`
-  .mapToolbarOverlay__button {
-    display: none;
-  }
-
-  ${({ maintainRatio }) =>
-    maintainRatio &&
-    css`
-      padding-top: calc(3 / 4 * 100%); /* 4:3 (standard) ratio */
-      position: relative;
-
-      @media only screen and (min-width: ${({ theme }) => theme.eui.euiBreakpoints.m}) {
-        padding-top: calc(9 / 32 * 100%); /* 32:9 (ultra widescreen) ratio */
-      }
-
-      @media only screen and (min-width: 1441px) and (min-height: 901px) {
-        padding-top: calc(9 / 21 * 100%); /* 21:9 (ultrawide) ratio */
-      }
-
-      .embPanel {
-        bottom: 0;
-        left: 0;
-        position: absolute;
-        right: 0;
-        top: 0;
-      }
-    `}
-`;
+const EmbeddableMapRatioHolder = styled('div', {
+  shouldForwardProp: (prop) => prop !== 'maintainRatio',
+})<EmbeddableMapProps>(({ maintainRatio, theme: { euiTheme } }) => ({
+  '& .mapToolbarOverlay__button': {
```

Comment:
nit: we can omit the self-reference

https://github.com/user-attachments/assets/55330a47-ce1e-4cbf-93d2-6648c26e5a03